docs/docsite/rst/network/user_guide/network_resource_modules.rst

```diff
@@ -53,7 +53,7 @@ The following table shows an example of how an initial resource configuration ch
 +-----------------------------------------+------------------------------------+-----------------------------------------+
 | Resource starting configuration         | task-provided configuration (YAML) | Final resource configuration on device  |
 +=========================================+====================================+=========================================+
-| .. code-block:: text                    |  .. code-block:: yaml              | *merged*                                |
+| .. code-block:: text                    |  .. code-block:: ini               | *merged*                                | 
```

Comment:
Avoid trailing whitespaces. Also, it's not ini, just like in many places Felix marked.
```suggestion
| .. code-block:: text                    |  .. code-block:: yaml               | *merged*                                |
```